src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -182,8 +183,10 @@ export abstract class GameWrapper<P extends Comparable> {
             isThereAPlayer: this.players[indexPlayer],
         } });
         if (this.players[indexPlayer].isPresent()) {
+            console.log('men it is not your turn')
```

Comment:
and women?!!

Reply:
did you ever see, only once, two women on this website :D ?

Reply:
j'suis certain que plus d'une femme a visité ce site oui !
Puis elles ont vu tes logs et se sont dites « ah non c'est un site pour homme » hélas !

Reply:
Il n'y a eu plus de une autre personne que moi sur ce site que deux fois, et jamais deux d'entre eux étaient des femmes !
Donc je ne crois pas mossieur !
Et tu part du principe que toutes les femmes qui ont visité ce site étaient des devs du coup, tu t'avances !